ANONYMITY_NETWORKS.md

```diff
@@ -222,3 +222,77 @@ leak information to hidden services if done improperly.
 At the current time, if users need to frequently make transactions, I2P/Tor
 will improve privacy from ISPs and other common adversaries, but still have
 some metadata leakages to unknown hidden service operators.
+
+# Example: Configure a remote, linux daemon that you can connect to via tor from a linux or mac laptop
```

Comment:
Another minor comment... "from a linux or mac laptop" The instruction doesn't exclude Win or BSD so why call out mac/linux?

Reply:
Makes sense.